src/utilities/sticky-manager/sticky-manager.ts

```diff
@@ -1,4 +1,4 @@
-import debounce from 'lodash/debounce';
+import throttle from 'lodash/throttle';
```

Comment:
Ahhh gotcha, I was wondering why we were doing the `debounce` equivalent instead of just using `throttle`, keeping the bundle down makes sense. I'll replace it:)